make_profiler/report/script.js

```diff
@@ -0,0 +1,157 @@
+function filterTarget() {
```

Comment:
this function has no input, no output, no comment on what it does and lives in file named "script". can it be a bit more descriptive?

Reply:
added comment for filtering